docs/competition/event/incident-management.md

```diff
@@ -0,0 +1,85 @@
+# SR2019 Incident Management Procedure
+
+This page is primarily for internal use. If you are a team competing in Student Robotics, you probably want our main website.
+
+This page details the procedures to be following in the event of an incident occuring during the competition. Incidents are split into 2 types; minor and major.
+
+What constitutes an incident is rather unpredictable, and as such is not fully defined. Common sense should prevail in determining the nature and severity of the incident.
+
+This procedure is superceded by any requirements of the venue, e.g. in the event of a fire follow the venues' fire evacuation procedure.
+
+# Minor Incidents
+
+A minor incident is anything that needs some kind of response but has minimal impact on the event.
+
+Examples may be;
+
+* A competitor has twisted their ankle on a stairwell,
+* Infrastructure cabling has become damaged and poses a hazard.
+
+Procedure here is simple;
+
+* Notify relevant coordinator,
+* Follow their instructions
+
+# Major Incidents
+
+A major incident is anything that causes a significant impact on the event as a whole.
+
+Examples might include;
+
+* Accident requiring significant first aid,
+* Battery failure renders access to an area unsafe
+
+As with minor incidents, the initial reaction should be to notify the relevant coordinator and follow their instructions. If first aid is required the First Aid Provider should __always__ be notified first.
+
+## Stopping the competition
+
+A major inicident may call for the competition to be stopped. This may be at the end of a current match or in the middle of a match.
+
+Due to the impact of stopping the competition, ability to do so is restricted to the following roles:
+
+* Health and Safety Cooordinator,
+* Event Coordinator,
+* Head Judge,
+* Majority decision from Trustees.
+
+Stopping the competition will be indicated by a call over the radio of "This is _role_; close the arena."
```

Comment:
I think we need a different magic phrase here, we use the term "close the arena" too much when we close it after tinker-time. Maybe "Halt the competition"?

Reply:
Perhaps. I might wait and see if any other suggestions come out of the woodwork.
Whatever it is, it needs to be something that doesn't immediately make anyone overhearing it panic, while being sufficiently clear as to the intent.

Reply:
Maybe it's not worth defining the exact message to be used. If we're defining this as 'stopping the competition', then simply saying 'stop the competition' would work, likely with a 'I repeat...' part to ensure it's heard correctly.

Alternatively, and in no way seriously, code words with specific intent also work, like say [pineapple](https://youtu.be/gcL8K0QqhJA?t=31).

Reply:
We want the wording to be reasonably specific to avoid another message being misinterpreted as stopping the competition.

I've changed the wording, and required repeating it.